docs/02-get-started/01-quick-install.md

```diff
@@ -17,11 +17,13 @@ To get started with Kyma, let's quickly install it with specific modules first.
 1. To provision a k3d cluster, run:
 
   ```bash
-  k3d cluster create --k3s-arg '--tls-san=host.docker.internal@server:*'
+  k3d cluster create kyma --kubeconfig-switch-context -p '80:80@loadbalancer' -p '443:443@loadbalancer' --k3s-arg '--disable=traefik@server:*' --k3s-arg '--tls-san=host.docker.internal@server:*'
```

Comment:
80 and 443 might be reserved ports. so they may not be possible to use

Reply:
okay, let's use 30080 and 30443 then. I'll update commands in API Gateway tutorials with another PR